dataEntry.py

```diff
@@ -231,6 +231,14 @@ def handle_inlines_checks(string):
     string = sub(r"basic %s save against your class DC" % SAVES, r"@Check[type:\1|dc:resolve(@actor.system.attributes.classDC.value)|basic:true] save against your class DC", string)
     string = sub(r"%s save against your class DC" % SAVES, r"@Check[type:\1|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC", string)
     
+    # Lores - capture 1-2 words between 'DC ##' and 'Lore'
+    def sluggify_lore(match):

```

Comment:
We actually don't need to sluggify lores, `@Check[type:Astronomy Lore]` works automatically and doesn't need a localized label. Unless that broke, In which case we should fix that as a bug.

We'd also want to move the function definition out away from the main method.

Reply:
Hadn't realized non-sluggified worked now. Will update!